molecularnodes/entities/molecule/base.py

```diff
@@ -167,6 +170,9 @@ def _read(
         file_path : Union[Path, io.BytesIO]
             The file path to the file which stores the atomic coordinates.
         """
+        if StructureDownloader.is_valid_url(file_path):
```

Comment:
Note: this largely replicates approach of mn.Molecule.fetch() so that could potentially be consolidated into mn.Molecule.load() the same way.